diff -urNad dynamips-0.2.6-RC4~/Makefile dynamips-0.2.6-RC4/Makefile
--- dynamips-0.2.6-RC4~/Makefile	2006-11-15 23:05:30.000000000 +0100
+++ dynamips-0.2.6-RC4/Makefile	2006-11-19 23:10:55.000000000 +0100
@@ -211,7 +211,7 @@
 .PHONY: clean
 clean:
 	$(RM) -f rom2c$(BIN_EXT) microcode_dump.inc asmdefs$(BIN_EXT) \
-	asmdefs.h $(C_OBJS) $(A_OBJS) $(PROG)
+	asmdefs.h $(C_OBJS) $(A_OBJS) $(PROG) nvram_export
 	$(RM) -f *~
 
 .PHONY: package
