diff -urNad dynamips-0.2.6-RC4~/Makefile dynamips-0.2.6-RC4/Makefile
--- dynamips-0.2.6-RC4~/Makefile	2006-11-20 00:04:39.000000000 +0100
+++ dynamips-0.2.6-RC4/Makefile	2006-11-20 00:05:37.000000000 +0100
@@ -200,7 +200,8 @@
 
 install: $(PROG) nvram_export
 	@echo "Installing"
-	install -d $(DESTDIR)/bin $(DESTDIR)/share/man/man1 $(DESTDIR)/share/man/man7 $(DESTDIR)/etc
+	install -d $(DESTDIR)/bin $(DESTDIR)/share/man/man1 $(DESTDIR)/share/man/man7
+#$(DESTDIR)/etc
 	install dynamips nvram_export   $(DESTDIR)/bin
 	install -m644 dynamips.1        $(DESTDIR)/share/man/man1
 	install -m644 nvram_export.1    $(DESTDIR)/share/man/man1
