Index: dynamips/Makefile
===================================================================
--- dynamips.orig/Makefile	2010-09-10 21:09:46.000000000 +0000
+++ dynamips/Makefile	2010-09-10 21:21:05.000000000 +0000
@@ -44,7 +44,6 @@
 	-DHAS_RFC2553=$(HAS_RFC2553) \
 	-DHAS_POSIX_MEMALIGN=$(HAS_POSIX_MEMALIGN)
 
-#PCAP_LIB=/usr/local/lib/libpcap.a
 PCAP_LIB=-lpcap
 
 ifeq ($(shell uname), FreeBSD)
@@ -56,7 +55,7 @@
 else
 ifeq ($(shell uname), Linux)
    PTHREAD_LIBS?=-lpthread
-#   PCAP_LIB=-lpcap
+   PCAP_LIB=-lpcap
    CFLAGS+=-I/usr/include -I. $(PTHREAD_CFLAGS)
    LIBS=-L/usr/lib -L. -ldl /usr/lib/libelf.a $(PTHREAD_LIBS)
    DESTDIR=/usr
