diff -urNad dynamips-0.2.7-0.2.8RC2~/Makefile dynamips-0.2.7-0.2.8RC2/Makefile
--- dynamips-0.2.7-0.2.8RC2~/Makefile	2008-09-28 22:42:39.000000000 +0200
+++ dynamips-0.2.7-0.2.8RC2/Makefile	2008-09-28 22:43:00.000000000 +0200
@@ -3,7 +3,19 @@
 
 # Replace x86 by amd64 for a build on x86_64.
 # Use "nojit" for architectures that are not x86 or x86_64.
+ifeq ($(shell dpkg-architecture -qDEB_HOST_ARCH),i386)
+DYNAMIPS_ARCH?=x86
+else
+ifeq ($(shell dpkg-architecture -qDEB_HOST_ARCH),lpia)
 DYNAMIPS_ARCH?=x86
+else
+ifeq ($(shell dpkg-architecture -qDEB_HOST_ARCH),amd64)
+DYNAMIPS_ARCH?=amd64
+else
+DYNAMIPS_ARCH?=nojit
+endif
+endif
+endif
 
 # Change this to 0 if your system doesn't support RFC 2553 extensions
 HAS_RFC2553?=1
