diff -urNad dynamips-0.2.7-0.2.8RC2~/utils.h dynamips-0.2.7-0.2.8RC2/utils.h
--- dynamips-0.2.7-0.2.8RC2~/utils.h	2007-10-17 07:35:50.000000000 +0200
+++ dynamips-0.2.7-0.2.8RC2/utils.h	2008-09-29 14:11:38.000000000 +0200
@@ -46,6 +46,8 @@
 #define ARCH_BYTE_ORDER ARCH_BIG_ENDIAN
 #elif defined(__sparc) || defined(__sparc__)
 #define ARCH_BYTE_ORDER ARCH_BIG_ENDIAN
+#elif defined(__hppa) || defined(__hppa__)
+#define ARCH_BYTE_ORDER ARCH_BIG_ENDIAN
 #elif defined(__alpha) || defined(__alpha__)
 #define ARCH_BYTE_ORDER ARCH_LITTLE_ENDIAN
 #elif defined(__i386) || defined(__i386__) || defined(i386)
