Description: libelf moved due to multiarch
Origin: vendor
Bug-Debian: http://bugs.debian.org/682412
Forwarded: no
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2012-09-03

--- a/Makefile
+++ b/Makefile
@@ -69,7 +69,7 @@
    PTHREAD_LIBS?=-lpthread
    PCAP_LIB=-lpcap
    CFLAGS+=-I/usr/include -I. $(PTHREAD_CFLAGS)
-   LIBS=-L/usr/lib -L. -ldl /usr/lib/libelf.a $(PTHREAD_LIBS)
+   LIBS=-L/usr/lib -L. -ldl -lelf $(PTHREAD_LIBS)
    DESTDIR=/usr
 else
 ifeq ($(shell uname -s), Darwin)
